src/app/users/AdminGuard.ts

```diff
@@ -5,32 +5,32 @@ import { Roles } from './roles'
 @Injectable()
 export class AdminGuard extends AuthenticatedGuard {
   override isAllowed() {
-    return Roles.admin
+    return true;//Roles.admin
```

Comment:
Why?

Reply:
An attempt to fix the issue I mentioned in another TODO comment that I forgot to remove before committing.